go-chaos/internal/pods.go

```diff
@@ -148,9 +148,9 @@ func (c K8Client) checkIfBrokersAreRunning() (bool, error) {
 
 	allRunning := true
 	for _, pod := range pods.Items {
-		if pod.Status.Phase != v1.PodRunning {
+		if pod.Status.ContainerStatuses[0].Ready != true { // assuming there is only one container
```

Comment:
Why was this necessary?

Reply:
The status of the Pod is Running even before it is ready. So checking the status is not enough.